crates/aptos-crypto/src/ed25519/ed25519_keys.rs

```diff
@@ -58,7 +58,7 @@ impl Ed25519PrivateKey {
 
     /// Private function aimed at minimizing code duplication between sign
     /// methods of the SigningKey implementation. This should remain private.
-    fn sign_arbitrary_message(&self, message: &[u8]) -> Ed25519Signature {
+    pub fn sign_arbitrary_message(&self, message: &[u8]) -> Ed25519Signature {
```

Comment:
Instead of doing this, change `aptos-move/framework/Cargo.toml` to import `aptos_crypto` with the `fuzzing` feature:

```
diff --git a/aptos-move/framework/Cargo.toml b/aptos-move/framework/Cargo.toml
index de1b1ad123..4519344909 100644
--- a/aptos-move/framework/Cargo.toml
+++ b/aptos-move/framework/Cargo.toml
@@ -36,7 +36,7 @@ tempfile = "3.3.0"
 tiny-keccak = { version = "2.0.2", features = ["keccak"] }

 aptos-aggregator = { path = "../../aptos-move/aptos-aggregator" }
-aptos-crypto = { path = "../../crates/aptos-crypto" }
+aptos-crypto = { path = "../../crates/aptos-crypto", features = ["fuzzing"] }
 aptos-module-verifier = { path = "../../aptos-move/aptos-module-verifier" }
 aptos-sdk-builder = { path = "../aptos-sdk-builder" }
 aptos-state-view = { path = "../../storage/state-view" }
```

Reply:
Any luck with this? We do not want make to expose `sign_arbitrary_message` as public by default...